### PR TITLE
README: config path

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Running the `exo config` command will guide you through the initial configuratio
 
 You can find your API credentials in the *Account* section of the [Exoscale Console](https://portal.exoscale.com/account/profile/api).
 
-The configuration file and all assets created during `exo` operations will be saved in the `.exoscale` folder inside your home directory.
+The configuration file and all assets created during `exo` operations will be saved in the `.config/exoscale` folder inside your home directory.
```
Comment (falzm) [inline README.md:32]: Well, it's a bit more complicated than that... In practice, it depends on the OS: on Linux and UNIX-like systems it's `$HOME/.config/exoscale`, on macOS it's `~/Library/Application Support/exoscale/`, on MS Windows I have no idea where... As if it weren't complicated enough, we _also_ support the legacy location `$HOME/.exoscale`. So if your change doesn't help everybody, I can't accept it as is I'm afraid ¯\\\_(ツ)_/¯

Comment (SebastienPittet) [reply README.md:32]: On Windows :

%USERPROFILE%\.exoscale\exoscale.toml

Comment (raphink) [reply README.md:32]: Oh well. I was just surprised because `exo` generated its conf in `~/.config/exoscale` on my machine. Thanks for the explanation.

Comment (falzm) [reply README.md:32]: @raphink agreed, this situation is not ideal. We should document this more extensively in the README, but in any case this PR won't cut it. Thank you for trying, though :)

**Comment by falzm** [comment]
FYI, README has been updated in a14117d.

**Comment by raphink** [comment]
Thanks!

**Pull request closed without merging**